functorch/test/test_pythonkey.py

```diff
@@ -74,6 +76,53 @@ def f(x):
         new_inp = torch.randn(3)
         self.assertEqual(fx_f(new_inp), f(new_inp))
 
+    def test_make_fx_model_fwd_bwd(self, device):
+        class Foo(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.linear = nn.Linear(5, 5)
+
+            def forward(self, x):
+                return self.linear(x).relu()
+
+        model = Foo()
+
+        def f(x, params):
+            out = stateless.functional_call(model, params, x).sum()
+            out.backward()
+            return list(params.values())
+        input = torch.randn(3, 5, requires_grad=True)
+        params = dict(model.named_parameters())
+        fx_f = make_fx(f)(input, params)
+        # TODO: what assert statement should we add here?
```

Comment:
You should run the traced graph with inputs and compare the output against running the original function.

Reply:
Done. `fx_f` may change the order of parameters, and may have a numerical difference, so I took that into consideration when comparing